src/custom_envs/register_envs.py

```diff
@@ -512,9 +512,9 @@ def variable_goal_env(**kwargs):
 def register_custom_test_envs():
     register(id="MoonlanderWorld-dodge-gaussian-hard-v0",
```

Comment:
```suggestion
    register(id="MoonlanderWorld-dodge-gaussian-hard-test-v0",
```

Ich würde die Envs mit Test benennnen. Auch wenn sie in einer extra Methode sind beugt das Verwechslungen vor :D